product.yml

```diff
@@ -706,6 +708,8 @@ bug_mapping:
       issue_component: Cloud Compute / Cloud Controller Manager
     ose-kubevirt-csi-driver-container:
       issue_component: Cloud Compute / KubeVirt Provider
+    ose-kubevirt-csi-driver-rhel8:
```

Comment:
Containers always have a `-container` suffix to their ART metadata file name.

Reply:
thanks. updated and pushed